rest/model/pod.go

```diff
@@ -88,8 +97,8 @@ func (p *APICreatePod) splitEnvVars() (envVars map[string]string, secrets map[st
 // APIPod represents a pod to be used and returned from the REST API.
 type APIPod struct {
 	ID                        *string                            `json:"id"`
-	Status                    *APIPodStatus                      `json:"status"`
-	Secret                    *string                            `json:"secret,omitempty"`
+	Status                    APIPodStatus                       `json:"status"`
+	Secret                    APIPodSecret                       `json:"secret,omitempty"`
```

Comment:
Should this be a pointer if we want to omit empty?

Reply:
I guess it could be a pointer but it probably doesn't matter that much since the `APIPodSecret` has only pointer fields. I removed omitempty from all the struct fields in `APIPod`.